packages/office-ui-fabric-react/src/components/Stack/StackItem/StackItem.styles.ts

```diff
@@ -10,7 +10,13 @@ const alignMap: { [key: string]: string } = {
   end: 'flex-end'
 };
 
-export const styles: IStackItemComponent['styles'] = (props, theme): IStackItemStylesReturnType => {
+const baseTokens: IStackItemComponent['tokens'] = {
+  margin: 0
+};
+
+export const StackItemTokens: IStackItemComponent['tokens'] = (props, theme): IStackItemTokenReturnType => [baseTokens];
```

Comment:
Does this need to be a function? This seems redundant with `baseTokens`. Is there a need for both? (I don't see baseTokens used anywhere else.)